### ISSUE TITLE
Remove odm_reference_document from metadata schema

### ISSUE THREAD

**Issue by acorbi** [OpenDevelopmentMekong/ckanext-odm_dataset#265]
if not used anymore, odm_reference_document field should be removed from schema.

**Comment by acorbi** [comment]
@prustar @DBishton I believe data entry staff has been using this field recently to enter related documents for other profiles than ELC, correct? Is it still needed? Once it is not used anymore, we can hide it from the end-user

**Comment by prustar** [comment]
@acorbi - They have been using it for current profile pages as well. Not sure if it's still necessary and vital to the way the profile pages are built. Perhaps @Huyeng has a better idea?

**Comment by acorbi** [comment]
@prustar @huyeng yeah I think this has been (and will be in the future) used for the logic of the profile pages. My suggestion is leave it as it is since the CKAN is going to be hidden ferom the end user
